config/config.env.example

```diff
@@ -0,0 +1,25 @@
+# Output and Cache directories
+OUTDIR=/usr/src/app/output
+CACHEDIR=/usr/src/app/cache
+
+# Internal endpoints
+URI_API_ENDP=https://identifier.translation.service/translate
+AUTH_API_ENDP=https://authentication.service/tokens
+
+# Login details for the tokens API
+URI_API_USER=admin_user@openbookpublishers.com
+URI_API_PASS=some_secret_password
+
+# Crossref account details
+XREF_USERNAME=accountusername
+XREF_PASSWORD=accountpassword
+XREF_DOI_PREFIX=10.45123
+XREF_START_DATE=2018-12-31
+
+# Miscellaneous
+URI_SCHEME=info:doi
+URI_STRICT=true
+REPROCESS_REPORTS=true
+JWT_DISABLED=False
+
+MODES=[{"measure":"https://metrics.operas-eu.org/citations/crossref/v1"}]
```

Comment:
Inverse the order: `crossref/citations`. Also, if you have an official description of this let me know and I'll add it to the website